db/connection.go

```diff
@@ -80,14 +78,23 @@ func (c *Connection) Migrate(ctx context.Context) {
 		log.Println("Table does not exist")
 		// insert data
 		result := c.DB.MustExecContext(ctx, videoInsert)
-		fmt.Println(result)
+		count, _ := result.RowsAffected()
+		if count > 0 {
+			log.Println("Inserted data")
+		}
 	} else {
-		row.Scan(&count)
+		err := row.Scan(&count)
```

Comment:
Is there a rows.Close() somewhere?

Reply:
single row so no need to close

Reply:
https://pkg.go.dev/database/sql#Row

Reply:
my bad didn't see it was a row not rows.